### PR TITLE
Updated to sylius/plugin-skeleton v1.3.6

### PR DESCRIPTION
**Pull request by anthid**

### CODE REVIEW

**Comment by mattagohni** [comment]
First of all, thanks for your work. Looks great so far. If the `phpstan`-issues could be easily resolved, it would be awesome if we get it in this PR done.

**Comment by anthid** [comment]
At first sight looks like *|null typecasting divergences between docblocks and signature. Will take a look later today.

**Comment by anthid** [comment]
I have fixed the easy ones. There is still a couple of `phpstan` errors pointing to Symfony methods that aren't that self-evident to me. I will take a look in the coming days, but of course you can take a look in the meantime. :smile:

File: phpstan.neon [hunk 1]
```diff
@@ -36,3 +37,8 @@ parameters:
         - '/Symfony\\Component\\ExpressionLanguage\\ParserCache\\ParserCacheInterface/'
         - '/Symfony\\Component\\ExpressionLanguage\\ParserCache\\ParserCacheAdapter/'
         - '/Class Symfony\\Component\\Config\\Definition\\Builder\\TreeBuilder does not have a constructor and must be instantiated without any parameters/'
+
+        # Symfony null typehints
+        - '/Cannot call method arrayNode\(\) on Symfony\\Component\\Config\\Definition\\Builder\\NodeParentInterface\|null/'
```
Comment (michz) [inline phpstan.neon:42]: I understand why you are doing this. For the long term I'd prefer to remove these here and instead "fix" the code that is causing problems. If there is an annotation that states "null ist possible" then I think we should handle the case. If in fact the null case is not possible, we can safely do a "if is null throw exception" handling.

**Review by michz — APPROVED** [review]

File: phpstan.neon (hunk 1 above)
Comment (mattagohni) [reply phpstan.neon:42]: I agree. I will open an issue for this, so we can look at it after this PR is merged.

**Review by mattagohni — APPROVED** [review]
👍 and thx again for your work.

File: phpstan.neon (hunk 1 above)
Comment (anthid) [reply phpstan.neon:42]: In this case I gained confidence after seeing that this class was already added to ignore by Sylius team. Also, the code where `phpstan` throws the error is a node configuration with fluent interface where `->end()` could be something or `null`. And checking for `null` here probably will mess the readability or involve deeper changes. But yes, I agree that supress an error is not fixing it. :smile:

**Pull request merged**